src/libraries/System.Linq.AsyncEnumerable/README.md

```diff
@@ -0,0 +1,18 @@
+# System.Linq.AsyncEnumerable
+
+Language-Integrated Query (LINQ) is the name for a set of technologies based on the integration of query capabilities directly into the C# language.
+
+Documentation can be found at https://learn.microsoft.com/dotnet/api/system.linq.
+
+This library provides an implementation of LINQ APIs for `IAsyncEnumerable<T>`.
+
+## Contribution Bar
+
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../../libraries/README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Linq+label%3A%22help+wanted%22+) issues.
+
+## Deployment
+
+System.Linq.AsyncEnumerable is a NuGet package that gets shipped with every release of .NET.
```

Comment:
I don't see the library shipping as a nuget package (IsPackable isn't set to true in the source project).

Reply:
If the library should ship inbox and as a package, consider mentioning the inbox nature here as well.